### DISCUSSION TITLE
关于专业版闪退下载最新版变成免费版的问题

### DISCUSSION THREAD

**Discussion by DANLIWANGYUAN** [getlantern/lantern-forum-cn#289] [Q&A]
业版闪退下载最新版变成免费版的问题有没有办法解决
![Uploading 948a1471b5f711384d0e1fccf344641.jpg…]()

**Comment by MaxMKMax** [comment]
@DANLIWANGYUAN 您好. 请问您可以试看重新登陆吗?

同时请您提供我账号邮箱和购买单据. 我帮您问看.

好消息，我们已经修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern
